src/ReactiveUI.Winforms/Properties/AssemblyInfo.cs

```diff
@@ -7,4 +7,4 @@
 using System.Runtime.CompilerServices;
 using System.Windows;
 
-[assembly: InternalsVisibleTo("ReactiveUI.Tests_Net45")]
+[assembly: InternalsVisibleTo("ReactiveUI.Tests_Net461")]
```

Comment:
I don't think this is actually used anywhere (other than in the API surface tests, which this *should* break)